### ISSUE TITLE
NC29RC4 no CPU and no Memory graphs shown

### ISSUE THREAD

**Issue by tflidd** [nextcloud/serverinfo#588]
### Steps to reproduce
1. Install latest NC29RC4 beta version and update apps
2. open admin-> system monitor
3.

### Expected behaviour
See cpu usage and memory usage

### Actual behaviour
field remain empty:
![image](https://github.com/nextcloud/serverinfo/assets/8111200/84c549d1-15d4-4686-bbb4-ea9b2033dae1)

On the same server setup, a NC 27.1.8, it is working as it should:
![image](https://github.com/nextcloud/serverinfo/assets/8111200/5391aa9d-e81e-4dc9-b847-bc217abe792d)

At this point https://github.com/nextcloud/serverinfo/blob/master/lib/SystemStatistics.php#L155, I was able to log the output of the values, they correspond to the cpu load values. But they are not shown on the interface.

### Server configuration

**Operating system**: FreeBSD 13.3

**Web server:** Nginx

**Database:** MariaDB

**PHP version:** 8.2

**Nextcloud version:** NC29RC4

**Where did you install Nextcloud from:**
manually

**List of activated apps:**

```
  - activity: 2.21.1
  - bookmarks: 13.1.3
  - calendar: 4.7.0-rc.1
  - checksum: 1.2.4
  - cloud_federation_api: 1.12.0
  - comments: 1.19.0
  - contacts: 6.0.0-rc.1
  - contactsinteraction: 1.10.0
  - cospend: 1.6.1
  - dashboard: 7.9.0
  - dav: 1.30.1
  - deck: 1.13.0-beta.1
  - end_to_end_encryption: 1.15.2
  - federatedfilesharing: 1.19.0
  - federation: 1.19.0
  - files: 2.1.0
  - files_accesscontrol: 1.19.0
  - files_automatedtagging: 1.19.0
  - files_downloadlimit: 2.0.0
  - files_external: 1.21.0
  - files_pdfviewer: 2.10.0
  - files_reminders: 1.2.0
  - files_sharing: 1.21.0
  - files_trashbin: 1.19.0
  - files_versions: 1.22.0
  - firstrunwizard: 2.18.0
  - flow_notifications: 1.9.0
  - forms: 4.2.3
  - integration_discourse: 2.0.6
  - logreader: 2.14.0
  - lookup_server_connector: 1.17.0
  - mail: 3.6.0-rc.3
  - news: 25.0.0-alpha5
  - nextcloud_announcements: 1.18.0
  - notifications: 2.17.0
  - oauth2: 1.17.0
  - password_policy: 1.19.0
  - photos: 2.5.0
  - polls: 7.0.3
  - privacy: 1.13.0
  - provisioning_api: 1.19.0
  - radio: 1.0.3
  - recommendations: 2.1.0
  - related_resources: 1.4.0
  - serverinfo: 1.19.0
  - settings: 1.12.0
  - sharebymail: 1.19.0
  - systemtags: 1.19.0
  - text: 3.10.0
  - theming: 2.4.0
  - twofactor_backupcodes: 1.18.0
  - twofactor_totp: 11.0.0-dev
  - updatenotification: 1.19.1
  - user_status: 1.9.0
  - viewer: 2.3.0
  - weather_status: 1.9.0
  - workflow_script: 1.14.0
  - workflowengine: 2.11.0

```

there is nothing in the logs, despite putting it in debug mode in `config.php`.

There is another user reporting the same issue on the forum:
https://help.nextcloud.com/t/third-rc-of-nextcloud-29/187628/5?u=tflidd

And the NC 27 system has the serverinfo app in version `1.17.0`

**Comment by kesselb** [comment]
Thanks for your report :+1:

**Comment by tflidd** [comment]
It is still broken, didn't the patch (https://github.com/nextcloud/serverinfo/pull/590) make it in the final release?

**Comment by kesselb** [comment]
The patch was merged early enough for the final release. 
Cache maybe?

**Comment by tflidd** [comment]
Yes, it was the cache. I'm a bit surprised the cache survived an update of the server (RC-> final version). Cache max-age was >180 days...

**Comment by Arveen205** [comment]
I also don't see the CPU and memory graphs under the system. I get the error: 'Connection to server lost.'"

NC 30.0.4
OS: Ubuntu 22.04
Apache 2.4
PHP 8.3

**Comment by tflidd** [comment]
Where is this error in the web interface?
Probably it is a different error. Perhaps with more details, community people can help you out on the forum help.nextcloud.com.

If you have specific errors and a hint that there is really a bug, you can open a new bug report.

**Comment by Arveen205** [comment]
The error occurred in the web interface, but it was resolved when I cleared the browser cache.
Thank you for your reply.
